steps/virustotal/get-domain/manifest.yaml

```
@@ -12,11 +12,11 @@ envs:
     description: "Your VirusTotal API key."
     type: "string"
     required: true
-  - name: URL
-    description: "URL to be analyzed."
+  - name: DOMAIN
+    description: "DOMAIN to be analyzed."
     type: "string"
     required: true
-    example: "http://www.somedomain.com/this/is/my/url"
+    example: "http://www.somedomain.com/this/is/my/domain"
 
 outputs:
```

As I see the domain object has a reputation property I want to expose that in the output so please add
```suggestion
outputs:
- name: reputation
    description: "The reputation score for the domain in question."
    type: "int"
    example: 13
```

---

done